src/runner.sh

```diff
@@ -110,10 +114,30 @@ while [[ $# -gt 0 ]]; do
   esac
 done
 
-Runner::loadTestFiles
 
-trap 'Console::renderResult '\
-'"$(State::getTestsPassed)" '\
-'"$(State::getTestsFailed)" '\
-'"$(State::getAssertionsPassed)" '\
-'"$(State::getAssertionsFailed)"' EXIT
+function Runner::loadTestFiles() {
+  if [ ${#_FILES[@]} -eq 0 ]; then
+    echo "Error: At least one file path is required."
```

Comment:
Should use text with a red background here too?

Reply:
Good idea, but I would suggest doing this in a follow up PR

Reply:
Ok, I will do this in this PR, @Tito-Kati 😝